docs/app/flutter/package/howToDevelopPackage.md

```
@@ -0,0 +1,93 @@
+  ```
+
+- **[캐럿 구문](https://dart.dev/tools/pub/dependencies#caret-syntax)을 이용한 범위 제약**: 최소 버전을 설정한다. 이는 해당 버전부터 다음 메이저 버전 이전까지의 모든 버전을 포함한다.
+
+  ```yaml
+  dependencies:
+    collection: "^5.4.0"
+  ```
+
+  이 구문은 위 **범위 제약 조건** 과 같은 의미를 가진다.
+
+상세 내용은 다음에 살펴볼 [패키지 버전 관리 가이드](#패키지-버전-관리-가이드)에서 확인하자.
```

Markdownlint (0.35.0)

Link fragments should be valid

(MD051, link-fragments)